aea/protocols/default/models.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
why `models.py`? In fact, there's no model. So better having another module for that?

Reply:
@marcofavorito agreed, how about `custom_types.py`?

Reply:
Ok, but it is not a standard module name

Reply:
Changed to `custom_types.py`.